admiral/pkg/clusters/serviceentry.go

```diff
@@ -810,6 +809,33 @@ func modifyServiceEntryForNewServiceOrPod(
 			ctxLogger.Warnf(common.CtxLogFormat, "getAllVSRouteDestinationsByCluster",
 				deploymentOrRolloutName, eventNamespace, sourceCluster,
 				"No RouteDestinations generated for VS based routing ")
+
+			sourceClusterLocality, err = getClusterRegion(remoteRegistry, sourceCluster, rc)
```

Comment:
https://github.com/istio-ecosystem/admiral/pull/403/files#diff-fcb33638fb967d38496fdd069b6db31d4d879df2a93e0d47791cc97cdc55948bR813-R817 can be out of this else, also https://github.com/istio-ecosystem/admiral/pull/403/files#diff-fcb33638fb967d38496fdd069b6db31d4d879df2a93e0d47791cc97cdc55948bR835-R837 seems to be duplicate and will be executed anyways

Reply:
thanks @vrushalijoshi , I think I rebased incorrectly. Thanks for catching this